omise-php/lib/omise/res/OmiseApiResource.php

```diff
@@ -246,7 +246,7 @@ private function genOptions($requestMethod, $userpwd, $params) {
       $options += array(CURLOPT_USERAGENT => $user_agent);
 
     // Also merge POST parameters with the option.
-    if(count($params) > 0) $options += array(CURLOPT_POSTFIELDS => http_build_query($params));
+    if(gettype($params) == "array" && count($params) > 0) $options += array(CURLOPT_POSTFIELDS => http_build_query($params));
```

Comment:
@shunsuke-h-opn can you limit to 120 chars?

https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=14&id=omise_omise-eccube&open=AYuNuLq40JTxPDheTq7z

Reply:
@danfowler 
Got it.
Wait for a little.

Reply:
@danfowler 
https://github.com/omise/omise-eccube/pull/14/commits/68bda52fb2b029ea4c8ea6f3ffada71050cc6ff6
Fixed it.